packages/fondue/src/components/RichTextEditor/Plugins/EmojiPlugin/EmojiPicker/EmojiPickerSearchAndClear.tsx

```diff
@@ -0,0 +1,30 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Icons } from '@components/RichTextEditor/components/Toolbar/Icons';
+import { cn } from '@udecode/cn';
+import { UseEmojiPickerType } from '@udecode/plate-emoji';
+
+export type EmojiPickerSearchAndClearProps = Pick<UseEmojiPickerType, 'i18n' | 'searchValue' | 'clearSearch'>;
+
+export function EmojiPickerSearchAndClear({ i18n, searchValue, clearSearch }: EmojiPickerSearchAndClearProps) {
+    return (
+        <>
+            <span className={cn('tw-absolute tw-left-2 tw-top-1/2 tw-z-10 tw-flex tw-h-4 tw-w-4 tw--translate-y-1/2')}>
+                <Icons.search className="tw-h-full" />
```

Comment:
This icon and the clear icon seem to be incorrect. The magnifying class is incorrect here and the clear icon should be 20px instead of 16px